### PR TITLE
Add JS HTTP API example

### PR DESCRIPTION
**Pull request by tomicvladan**

### CODE REVIEW

File: js-http/fairos-http-client.js [hunk 1]
```diff
@@ -0,0 +1,623 @@
+// Let us define properties needed for API calls
+var username = "user_"+(Date.now() / 1000).toFixed(0)
+var password = "159263487"
+var podName = "pod1"
+var host = "https://localhost:9090/v1/";
```
Comment (asabya) [inline js-http/fairos-http-client.js]: for me `https` does not work on localhost
```
fairos-http-client.js:84 GET https://localhost:9090/v1//user/present?user_name=user_1643078197 net::ERR_SSL_PROTOCOL_ERROR
```

File: js-http/fairos-http-client.js [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+        },
+        body: JSON.stringify(data),
+        credentials: "include"
+    });
+}
+
+apiTest();
+
+function handleResponse(response) {
+    console.log(response);
```
Comment (asabya) [inline js-http/fairos-http-client.js:529]: After login cookie is not being set and sent with the following requests. thus seeing the following responses.
image

File: js-http/fairos-http-client.js (hunk 1 above)
Comment (asabya) [reply js-http/fairos-http-client.js]: The trailing slash seems to be causing some issues aswel

Comment (tomicvladan) [reply js-http/fairos-http-client.js]: Yes, probably that caused the cookie issue as well. I wasn't able to test it on localhost.

File: js-http/fairos-http-client.js (hunk 2 above)
Comment (asabya) [reply js-http/fairos-http-client.js:529]: This is related to setting cookieDomain to "localhost" from server side.

File: js-http/fairos-http-client.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 var username = "user_"+(Date.now() / 1000).toFixed(0)
 var password = "159263487"
 var podName = "pod1"
-var host = "https://localhost:9090/v1/";
```
Comment (tfius) [inline js-http/fairos-http-client.js]: const ?

Comment (tomicvladan) [reply js-http/fairos-http-client.js]: Yes, it makes more sense to use const instead of var.

**Review by asabya — APPROVED** [review]
LGTM

**Pull request merged**